src/pages/Home.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import hero from '../assets/hero.svg';
+import {MdOutlineDoubleArrow} from 'react-icons/md';
+import { categories } from '../backend/db/categories';
+import { CategoryCard } from '../components';
+import { Link } from 'react-router-dom';
```

Comment:
use [absolute path](https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i#:~:text=Absolute%20Path%20is%20when%20you,located%20in%20the%20project%20structure.)

Reply:
As it's not deeply nested so I think I can keep it as it is for now